### PR TITLE
Update login style

### LINKED ISSUE #1148: Login page cannot shrink
The LH side of the login has a min width. can we make it more responsive please?

### PR DESCRIPTION
**Pull request by mark-prins**
Fixes #1148 

Made the font sizes responsive - after realising that simply making font sizes `rem`s does not help with this little problem. We can change them later to rems, that's a different concern to this problem

### CODE REVIEW

File: packages/common/src/intl/locales/en/app.json [hunk 1]
```diff
@@ -56,7 +56,7 @@
   "cmdk.goto-reports": "Go to: Reports",
   "cmdk.goto-master-lists": "Go to: Master Lists",
   "cmdk.placeholder": "Type a command or search",
-  "login.heading": "Simple. Powerful. Pharmaceutical Management.",
+  "login.heading": "Simple.\nPowerful.\nPharmaceutical\nManagement.",
```
Comment (mark-prins) [inline packages/common/src/intl/locales/en/app.json:59]: pleased to find a way around the line wrapping concern 😁

File: packages/common/src/styles/theme.ts [hunk 2]
```diff
@@ -193,7 +193,7 @@ export const themeOptions = {
     },
     body2: { color: '#555770', fontSize: 12, fontWeight: 500 },
     fontFamily: 'InterVariable',
-    login: { color: '#fafafa', fontSize: 20, fontWeight: 600 },
+    login: { color: '#fafafa' },
```
Comment (mark-prins) [inline packages/common/src/styles/theme.ts:196]: these are not used in any of the currently defined override stylesheets

File: packages/host/src/components/Login/LoginLayout.tsx [hunk 3]
```diff
@@ -47,10 +47,16 @@ export const LoginLayout = ({
...
+                xs: '20px',
+                sm: '32px',
+                md: '38px',
+                lg: '48px',
+                xl: '64px',
+              },
               fontWeight: 'bold',
               lineHeight: 'normal',
-              maxWidth: '525px',
+              whiteSpace: 'pre-line',
```
Comment (mark-prins) [inline packages/host/src/components/Login/LoginLayout.tsx:59]: break lines on the line breaks

File: packages/host/src/components/Login/LoginLayout.tsx [hunk 4]
```diff
@@ -47,10 +47,16 @@ export const LoginLayout = ({
...
             sx={{
               color: theme => theme.typography.login.color,
-              fontSize: '64px',
+              fontSize: {
+                xs: '20px',
+                sm: '32px',
+                md: '38px',
+                lg: '48px',
+                xl: '64px',
+              },
```
Comment (CarlosNZ) [inline packages/host/src/components/Login/LoginLayout.tsx:56]: The heading font is now a lot smaller on my "normal" view than it was before -- is that intentional?

Previous:
Screen Shot 2022-04-06 at 8 17 04 PM

Now:
Screen Shot 2022-04-06 at 8 16 47 PM

This seems fine on my screen at all widths:
```
fontSize: {
                xs: '38px',
                sm: '48px',
                md: '64px',
                lg: '64px',
                xl: '64px',
              },
```

File: packages/common/src/intl/locales/en/app.json (hunk 1 above)
Comment (CarlosNZ) [reply packages/common/src/intl/locales/en/app.json:59]: This is very strange -- cos regardless of whether you have the line breaks in the string, the rendered HTML is still just:
Screen Shot 2022-04-06 at 8 37 03 PM
Not a `<br\>` to be seen! 😲

Anyway -- it works so...

File: packages/host/src/components/Login/LoginLayout.tsx [hunk 5]
```diff
@@ -59,9 +65,15 @@ export const LoginLayout = ({
         <Box style={{ marginTop: 45 }}>
           <Typography
             sx={{
-              fontSize: theme => theme.typography.login.fontSize,
+              fontSize: {
+                xs: '10px',
+                sm: '12px',
+                md: '14px',
+                lg: '16px',
+                xl: '20px',
```
Comment (CarlosNZ) [inline packages/host/src/components/Login/LoginLayout.tsx:73]: Similarly, this seems fine?
```
fontSize: {
                xs: '14px',
                sm: '16px',
                md: '20px',
                lg: '20px',
                xl: '20px',
              },
```

**Review by CarlosNZ — APPROVED** [review]
Nice. Just a small comment about the new font sizes.

File: packages/common/src/intl/locales/en/app.json (hunk 1 above)
Comment (mark-prins) [reply packages/common/src/intl/locales/en/app.json:59]: super weird.. 🤷

File: packages/host/src/components/Login/LoginLayout.tsx (hunk 4 above)
Comment (mark-prins) [reply packages/host/src/components/Login/LoginLayout.tsx:56]: ah. thanks - no, not intentional.

**Comment by mark-prins** [comment]
thanks - I've upped the font sizes 😁

**Pull request merged**